### ISSUE TITLE
vpncmd unable to create nic on Ubuntu 18.04

### ISSUE THREAD

**Issue by adamryczkowski** [SoftEtherVPN/SoftEtherVPN#926]
After setting up the server I try to connect to it. I followed the tutorial, installed the Linux client, ran `sudo vpnclient start`, then connected to it with `vpncmd`:

```
$ sudo ./vpncmd
vpncmd command - SoftEther VPN Command Line Management Utility
SoftEther VPN Command Line Management Utility (vpncmd command)
Version 4.29 Build 9680   (English)
Compiled 2019/02/28 19:22:54 by yagi at pc33
Copyright (c) SoftEther VPN Project. All Rights Reserved.

By using vpncmd program, the following can be achieved. 

1. Management of VPN Server or VPN Bridge 
2. Management of VPN Client
3. Use of VPN Tools (certificate creation and Network Traffic Speed Test Tool)

Select 1, 2 or 3: 2

Specify the host name or IP address of the computer that the destination VPN Client is operating on. 
If nothing is input and Enter is pressed, connection will be made to localhost (this computer).
Hostname of IP Address of Destination: 

Connected to VPN Client "localhost".

VPN Client>niccreate vpn0
NicCreate command - Create New Virtual Network Adapter
Error occurred. (Error code: 31)
Installation of the Virtual Network Adapter device driver failed.
VPN Client>

```

I also tried the following:

```
sudo modprobe tunnel4
sudo modprobe ip_tunnel
sudo modprobe tap
sudo modprobe tun
sudo ./vpnstart stop
sudo ./vpnstart start
sudo ./vpncmd
```

with the exactly the same result. 

-----

Interestingly, I've never found the confirmation that SoftEther client is actually supported on Ubuntu 18.04. If it is not, then please ignore this issue and kindly update the documentation.

**Comment by adamryczkowski** [comment]
Possible duplicate:  https://github.com/SoftEtherVPN/SoftEtherVPN/issues/148

**Comment by adamryczkowski** [comment]
I guess the problem is a really long and resolved one, see e.g. https://www.raspberrypi.org/forums/viewtopic.php?t=136232 . It makes me wonder if the SoftEther client was *ever* compatible with recent Ubuntu-based Linux.

**Comment by adamryczkowski** [comment]
Disabling apparmor does not help.

**Comment by oranav** [comment]
Run:
```
sudo ./vpnclient stop
ps -fade | grep vpnclient
```
And make sure no `vpnclient` service is running (`sudo kill` it if necessary).
Then make sure you run `vpnclient` as root:
```
sudo ./vpnclient start
````
And try to recreate the NIC using `vpncmd`. Works for me.

**Comment by adamryczkowski** [comment]
Yes, it works! Problem solved.

**Comment by chipitsine** [comment]
can you help with improving documentation ?

**Comment by adamryczkowski** [comment]
With pleasure. How can I start? I can't see a documentation repo.

**Comment by dp540788912** [comment]
@oranav 
Senpai, Why are you so smart? thank you very much!!!!!

**Comment by fti-cneve** [comment]
@all @chipitsine @dp540788912 @oranav @adamryczkowski 
Is this problem fixed in the newest kernel, such as 6.1.X?
